scripts/postbuild.js

```diff
@@ -0,0 +1,59 @@
+const path = require('node:path');
+const fs = require('node:fs/promises');
+
+// Copy /myPath.html to /myPath/index.html
+// This ensures both URL patterns work: /myPath and /myPath/
+async function generateIndexHtmlFiles(outDir) {
+	console.log('Post-build: Creating index.html files from hoisted pages...');
+
+	// Walk through all directories recursively
+	async function* walkDirs(dir) {
+		const dirents = await fs.readdir(dir, { withFileTypes: true });
+		for (const dirent of dirents) {
+			if (dirent.isDirectory()) {
+				const res = path.resolve(dir, dirent.name);
+				yield res;
+				yield* walkDirs(res);
+			}
+		}
+	}
+
+	const processedFiles = [];
+
+	for await (const dirPath of walkDirs(outDir)) {
+		// Check if there's already an index.html in this directory
+		const indexPath = path.join(dirPath, 'index.html');
+		try {
+			await fs.stat(indexPath);
```

Comment:
Should use `fs.access()` instead if you don't actually care for the result of `stat`. `access` is faster